templates/partials/button.html

```diff
@@ -0,0 +1,90 @@
+{% set disabled = disabled_status|default('') %}
+
+{% set button_library = {
```

Comment:
defining all these extra variables in this dictionary seems unnecessarily complicated. Can we just stick to "class" and "content" like we did in https://github.com/DACCS-Climate/marble-website/blob/main/templates/partials/button.html

Reply:
Replaced most variables with "content", removed extra display conditions.